ChangeLog

```diff
@@ -1,4 +1,4 @@
-	* drop support for windows compilers without std::wstring
```

Comment:
This change is no longer correct. I would suggest you squash and rebase your commits on top of master, and while doing so, drop both of the merge commits